Makefile

```diff
@@ -595,8 +597,12 @@ release-manifests: $(BUILD_DIR) $(MANIFEST_DIR) $(KUSTOMIZE) $(STAGE)-flavors ##
 	cp metadata.yaml $(MANIFEST_DIR)/metadata.yaml
 
 .PHONY: release-flavors ## Create release flavor manifests
-release-flavors: $(RELEASE_DIR)
-	$(MAKE) generate-flavors FLAVOR_DIR=$(RELEASE_DIR)
+release-flavors: $(KUSTOMIZE) $(addprefix release-flavors-, main)
```

Comment:
I think this breaks oure release GitHub action?

Reply:
Yup. I think I would stay away from dev-flavors and that overrides dir :D

Reply:
Maybe let's preserve whatever that dev workflow is (that we're not really using because of tilt) and have our own thing for e2e